test/extended/extended_test.go

```diff
@@ -0,0 +1,71 @@
+package extended
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
+	. "github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+	"github.com/golang/glog"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/gomega"
+)
+
+var (
+	reportDir = flag.String("report-dir", "", "Path to the directory where the JUnit XML reports should be saved. Default is empty, which doesn't generate these reports.")
+)
+
+// init initialize the extended testing suite.
+// You can set these environment variables to configure extended tests:
+// MASTER_ADDR - The host, or apiserver to connect to
```

Comment:
This should be unnecessary if we require a `KUBECONFIG` which seems more appropriate.